addons/mail/static/src/components/composer/composer.xml

```diff
@@ -154,7 +154,7 @@
                     <t t-else=""><i class="fa fa-paper-plane-o"/></t>
                 </button>
             </t>
-            <t t-if="!messaging.device.isMobile and props.hasDiscardButton">
+            <t t-if="!messaging.device.isMobile and composerView.hasDiscardButton">
```

Comment:
Shouldn't the condition on `messaging.device.isMobile` be part of the compute of `hasDiscardButton`?